src/gsy_e/models/strategy/external_strategies/load.py

```diff
@@ -314,9 +315,14 @@ def _bid_aggregator(self, arguments: Dict) -> Dict:
                     raise OrderCanNotBePosted("The load did not consume to little energy, "
                                               "settlement bid can not be posted.")
                 required_energy_kWh = self.state.get_unsettled_deviation_kWh(market.time_slot)
-            else:
+            elif self.area.is_market_future(market.id):
+                required_energy_kWh = self.state.get_energy_requirement_Wh(
+                    str_to_pendulum_datetime(arguments["time_slot"])) / 1000.
+            elif self.area.is_market_spot(market.id):
                 required_energy_kWh = (
                         self.state.get_energy_requirement_Wh(market.time_slot) / 1000.)
+            else:
+                raise OrderCanNotBePosted("Market can not be posted to.")
```

Comment:
Nit:
```suggestion
                raise OrderCanNotBePosted("The order cannot be posted on the market.")
```
It might even help to add a debug log, for example:
```suggestion
                logger.debug("The order cannot be posted on the market. (arguments: %s, market_id: %s"), arguments, market.id)
                raise OrderCanNotBePosted("The order cannot be posted on the market.")
```
This was we would be able to debug production if needed.

Reply:
Great Thanks! This error message was a placeholder and I wanted to improve it before pushing, I forgot unfortunately. Will go for your suggestion.